### DISCUSSION TITLE
Ability to filter commit by author?

### DISCUSSION THREAD

**Discussion by atulpatildbz** [jesseduffield/lazygit#2140] [Q&A]
I work on a huge project. Containing code from multiple teams which most of the time isn't useful for me. Therefore it's often more useful to just view my team's commits instead of all the developers. Is there a way to achieve this?

**Comment by mark2185** [comment]
Huh, that's currently not a feature. We only have "filter by path", by pressing `ctrl+s` in the `Commits` panel.

But I guess we could put such filters behind a menu. Would you maybe be interested in taking a shot at implementing this?

**Reply by atulpatildbz** [reply]
Sure! Currently I'm occupied with something else, but I might give it a shot next week.
Also, I'm not very familiar with Go so it might take me a little longer to implement, but I'll try.

**Reply by mark2185** [reply]
Take your time! Head over to our slack at `lazygit.slack.com` if you have any questions. 

Here's a [somewhat lengthy video](https://www.youtube.com/watch?v=kNavnhzZHtk) of adding a "reset author" feature to lazygit which should help you with getting a feel of what's what in the project.
